src/AccessibilityInsights.CommonUxComponents/Controls/ProgressRingControl.xaml.cs

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Windows.Controls;
```

Comment:
>System [](start = 6, length = 6)

Please sort the using blocks alphabetically

Reply:
Sorted